Resources/public/js/adapter/fancytree.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * A tree browser adapter for the Fancytree library.
+ *
+ * @author Wouter J <wouter@wouterj.nl>
+ * @see https://github.com/mar10/fancytree
+ */
+var FancytreeAdapter = function (data_url) {
+    if (!window.jQuery || !jQuery.fn.fancytree) {
+        throw 'The FancytreeAdapter requires both jQuery and the FancyTree library.';
+    }
+
+    var requestNodeToFancytreeNode = function (requestNode) {
+        var fancytreeNode = {
+            // fixme: remove ugly replacing by just not putting stuff like this in the JSON response
+            title: requestNode.data.replace(' (not editable)', ''),
```

Comment:
@dbu dan and I are planning to use the ResourceRestBundle as back-end for the TreeBrowserBundle and remove all back-end Tree stuff from the TreeBrowserBundle. That's why I just did this not so nice compability layer.